test/sagemaker_tests/tensorflow/tensorflow2_training/resources/smmodelparallel/smdataparallel_smmodelparallel_mnist_script_mode.sh

```diff
@@ -4,5 +4,6 @@
 
```

Comment:
This is a Tensorflow file, why have we made change to this?

Reply:
This change is also suggested by SMDDP team. The reason is to update the single node launcher that being used in both TF and PT tests.

Reply:
Thanks a lot!